### PR TITLE
Add firefox/developer.ftl

### PR DESCRIPTION
**Pull request by alexgibson**
Ref: https://github.com/mozilla/bedrock/pull/9793

### CODE REVIEW

File: en/firefox/developer.ftl [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+firefox-developer-firefox-has-been-rebuilt = { -brand-name-firefox } has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
+firefox-developer-congrats-you-now-have-firefox = Congrats. You now have { -brand-name-firefox-quantum }: { -brand-name-developer-edition }.
+firefox-developer-this-isnt-just-an-update = This isn’t just an update. This is { -brand-name-firefox } Quantum: A brand new { -brand-name-firefox } that has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
+firefox-developer-welcome-to-firefox-browser = Welcome to { -brand-name-firefox-browser } { -brand-name-developer-edition }
+firefox-developer-made-for-developers = The browser made for developers
+firefox-developer-all-the-latest = All the latest developer tools in beta, plus <strong>experimental features</strong> like the Multi-line Console Editor and WebSocket Inspector.
+firefox-developer-a-separate-profile = A <strong>separate profile and path</strong> so you can easily run it alongside Release or Beta { -brand-name-firefox }.
```
Comment (peiying2) [inline en/firefox/developer.ftl:84]: Do we make `Beta` a brand name item in this case?

File: en/firefox/developer.ftl [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+firefox-developer-firefox-developer-edition-sends = { -brand-name-firefox-developer-edition } automatically sends feedback to { -brand-name-mozilla }.
+firefox-developer-download-the-firefox-browser = Download the { -brand-name-firefox } browser made for developers
+firefox-developer-welcome-to-the-all-new = Welcome to the all-new { -brand-name-firefox-quantum }: { -brand-name-developer-edition }
+firefox-developer-firefox-has-been-rebuilt = { -brand-name-firefox } has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
+firefox-developer-congrats-you-now-have-firefox = Congrats. You now have { -brand-name-firefox-quantum }: { -brand-name-developer-edition }.
+firefox-developer-this-isnt-just-an-update = This isn’t just an update. This is { -brand-name-firefox } Quantum: A brand new { -brand-name-firefox } that has been rebuilt from the ground-up to be faster, sleeker, and more powerful than ever.
```
Comment (peiying2) [inline en/firefox/developer.ftl:80]: Let's make it  { -brand-name-firefox-quantum }

**Review by peiying2 — CHANGES_REQUESTED** [review]
Overall looks good, two brand-name related issues to check

**Comment by alexgibson** [comment]
@peiying2 updated thank you

**Review by peiying2 — APPROVED** [review]
Thank you for the fixes. LGTM.

**Pull request merged**